vcsclient/bitbucketcloud.go

```diff
@@ -782,3 +788,11 @@ func getBitbucketCloudRepositoryVisibility(repo *bitbucket.Repository) Repositor
 	}
 	return Public
 }
+
+func splitWorkSpaceAndOwner(name string) (string, string) {
```

Comment:
Let's add a description of this function. What is the expected format and the result